### PR TITLE
Added tests to Math and Number

### PR DESCRIPTION
**Pull request by jlumme**

### CODE REVIEW

File: test/Number.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Prepare chai
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('Number', function() {
+  describe('static isNumber', function() {
```
Comment (marverix) [inline test/Number.js]: If you are describing static functions then please use `#`. E.g. `#isNumber`. When you are describing instance method then use dot - `.something`

File: test/Number.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    it('undefined is not a number', function() {
+      expect(Number.isNumber('undefined')).to.be.false;
+    });
+
+    it('1 is a number', function() {
+      expect(Number.isNumber(1)).to.be.true;
+    });
+  });
+});
```
Comment (marverix) [inline test/Number.js]: Each text file should end with new line. Please read more about it here: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: test/Number.js (hunk 1 above)
Comment (jlumme) [reply test/Number.js]: Thanks, I'll fix it.

File: test/Number.js (hunk 2 above)
Comment (jlumme) [reply test/Number.js]: Thanks for the info, I added a new line to end of file

**Comment by marverix** [comment]
Thanks for contribution! +1

**Pull request merged**